components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -80,9 +94,11 @@
     "webRequest",
     "*://*/*",
     "chrome://favicon/*",
+    "webRequestBlocking",
+    "unlimitedStorage",
     "<all_urls>"
   ],
-  "content_security_policy": "default-src 'self'; font-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'; img-src 'self' data: chrome://favicon/; connect-src https://pcdn.brave.com https://brave-today-cdn.brave.com https://pcdn.bravesoftware.com https://brave-today-cdn.bravesoftware.com https://pcdn.brave.software https://brave-today-cdn.brave.software https://api.twitter.com 'self';",
+  "content_security_policy": "default-src 'self' ; font-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'; img-src 'self' data: chrome://favicon/; connect-src 'self' data:;",
```

Comment:
i don't think you need the space after 'self'.

Reply:
fixed